src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -45,15 +45,26 @@ class ValidateActor(wdlSource: WdlSource, wdlJson: WdlJson)
       // NOTE: self shuts down when the parent PerRequest shuts down
   }
 
+  private def validateInputs(namespaceWithWorkflow: NamespaceWithWorkflow, maybeWorkflowInputs: Option[WdlJson]): Future[Option[WorkflowCoercedInputs]] = {
+    maybeWorkflowInputs match {
+      case Some(wi) =>
+        for {
+          inputs <- Future(wi.parseJson).map(_.asJsObject.fields)
+          coercedInputs <- Future.fromTry(namespaceWithWorkflow.coerceRawInputs(inputs))
+        } yield Some(coercedInputs)
+      case None => Future.successful(None)
+    }
+  }
```

Comment:
Looks like it could be `Future[Unit]`, the caller doesn't seem to care about the returned value.